05week/spaceTravelToMars.js

```diff
@@ -11,6 +11,51 @@ let jobTypes = {
 
 // Your code here
 
+class Ship {
+  constructor(name, type, ability){
+    this.name = name;
+    this.type = type;
+    this.ability = ability;
+    this.crewMems = [];
+  }
+    missionStatement() {
+      for (var x = 0; x < this.crewMems.length; x++) {
+        let crewPerson = this.crewMems[x].job;
+        if (jobTypes[crewPerson] === this.type) {
+          return this.ability;
+          } else {
+          return "Can't perform a mission yet.";
```

Comment:
the return for "Can't perform a mission yet." need to happen after the for loop.
The way you have implemented it, requires that the first crew member in the list match the ships ability.